march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -411,7 +411,7 @@ void MarchHardwareInterface::updateIMotionCubeState()
     imc_state_pub_->msg_.detailed_error_description[i] = imc_state.detailedErrorDescription;
     imc_state_pub_->msg_.motion_error_description[i] = imc_state.motionErrorDescription;
     imc_state_pub_->msg_.motor_current[i] = imc_state.motorCurrent;
-    imc_state_pub_->msg_.motor_voltage[i] = imc_state.motorVoltage;
+    imc_state_pub_->msg_.IMC_voltage[i] = imc_state.IMCVoltage;
```

Comment:
In the march repository the message is defined to have a motor_voltage component, so you would have to change the message structure if you want this.

Reply:
I actually already did this, because I saw my build failing. However I was not aware that I needed to make a seperate PR for the march repo, so I will do this asap!